### DISCUSSION TITLE
Problem with on().subscribe(). Listens only to one table

### DISCUSSION THREAD

**Discussion by T0nyBoy** [supabase/supabase#6884] [Questions] [answered]
I am trying to create a script that will listen to a table. I have created one and it runs smoothly so I copied it and renamed the table that it should listen to and I thought I would get some results but even though it runs, it doesnt return a thing.
the basic script is:
```
try {
  const mySubscription = supabase
    .from("table1")
    .on("*", (payload) => {
      console.log(payload.new);
    .subscribe();
} catch (e) {
  console.log(`[!] Error: ${e}`);
}
```
It prints the normal response.
But when I change it to table2 it just runs and does nothing.
The thing is that I doublicated the table1 and renamed it to table2 but again nothing.
Then I tried to listen to all the tables with .from("*") and again it listens to only table1.
I checked the logs and I see that from the API logs the requests to table1 add 2 items to the logs, one HEAD and another OPTIONS and they both have status 200. To all the other tables and table2 I get a GET with status 101. Also, the client from the HEAD and OPTIONS is https://app.supabase.io and for the other tables' GET the client is supabase-js/1.35.3 . 
Any help or idea?
Thank you! :)

**Comment by GaryAustin1** [comment, marked as answer]
Do you have replication turned on for the new table?
Do you have RLS on and not setup for selects?
How did you duplicate the table?  Try and set up another table thru the ui if you did some other way to rule that out.

**Reply by T0nyBoy** [reply]
replication was the culprit... :) thank you very much!
